contracts/governance/Staking/Staking.sol

```diff
@@ -20,14 +20,44 @@ contract Staking is IStaking, WeightedStaking {
 		address stakeFor,
 		address delegatee
 	) public {
+		_stake(msg.sender, amount, until, stakeFor, delegatee);
```

Comment:
mark `stake` as external and call `_stake` internally instead

Reply:
done